sygnal/apnspushkin.py

```diff
@@ -311,8 +311,9 @@ async def _dispatch_notification_unlimited(
             if device.data:
                 default_payload = device.data.get("default_payload", {})
                 if not isinstance(default_payload, dict):
-                    log.error(
-                        "default_payload is malformed, this value must be a dict."
+                    logger.warning(
```

Comment:
I think we want to continue using `log` instead of `logger` here too?

Reply:
Yes I missed this, have fixed.